projects/kubernetes-sigs/cluster-api/build/create_manifests.sh

```diff
@@ -34,41 +34,28 @@ KUBE_RBAC_PROXY_LATEST_TAG=latest
 
 make set-manifest-image \
     MANIFEST_IMG=$CAPI_REGISTRY_PREFIX/cluster-api-controller MANIFEST_TAG=$IMAGE_TAG \
-    TARGET_RESOURCE="./config/manager/manager_image_patch.yaml"
+    TARGET_RESOURCE="./config/default/manager_image_patch.yaml"
 
 # Set the kubeadm bootstrap image to the production bucket.
 make set-manifest-image \
     MANIFEST_IMG=$CAPI_REGISTRY_PREFIX/kubeadm-bootstrap-controller MANIFEST_TAG=$IMAGE_TAG \
-    TARGET_RESOURCE="./bootstrap/kubeadm/config/manager/manager_image_patch.yaml"
+    TARGET_RESOURCE="./bootstrap/kubeadm/config/default/manager_image_patch.yaml"
 
 # Set the kubeadm control plane image to the production bucket.
 make set-manifest-image \
     MANIFEST_IMG=$CAPI_REGISTRY_PREFIX/kubeadm-control-plane-controller MANIFEST_TAG=$IMAGE_TAG  \
-    TARGET_RESOURCE="./controlplane/kubeadm/config/manager/manager_image_patch.yaml"
+    TARGET_RESOURCE="./controlplane/kubeadm/config/default/manager_image_patch.yaml"
 
-make set-manifest-image \
-    MANIFEST_IMG=${IMAGE_REPO}/brancz/kube-rbac-proxy MANIFEST_TAG=$KUBE_RBAC_PROXY_LATEST_TAG \
-    TARGET_RESOURCE="./config/manager/manager_auth_proxy_patch.yaml"
-
-make set-manifest-image \
-    MANIFEST_IMG=${IMAGE_REPO}/brancz/kube-rbac-proxy MANIFEST_TAG=$KUBE_RBAC_PROXY_LATEST_TAG \
-    TARGET_RESOURCE="./bootstrap/kubeadm/config/manager/manager_auth_proxy_patch.yaml"
-
-make set-manifest-image \
-    MANIFEST_IMG=${IMAGE_REPO}/brancz/kube-rbac-proxy MANIFEST_TAG=$KUBE_RBAC_PROXY_LATEST_TAG \
-    TARGET_RESOURCE="./controlplane/kubeadm/config/manager/manager_auth_proxy_patch.yaml"
-
-make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./config/manager/manager_pull_policy.yaml"
-make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./bootstrap/kubeadm/config/manager/manager_pull_policy.yaml"
-make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./controlplane/kubeadm/config/manager/manager_pull_policy.yaml"
+make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./config/default/manager_pull_policy.yaml"
+make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./bootstrap/kubeadm/config/default/manager_pull_policy.yaml"
+make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./controlplane/kubeadm/config/default/manager_pull_policy.yaml"
 
 ## Build the manifests
 make release-manifests
 
 ## Build the development manifests
 make -C test/infrastructure/docker set-manifest-image \
 MANIFEST_IMG=$CAPI_REGISTRY_PREFIX/capd-manager MANIFEST_TAG=$IMAGE_TAG
-sed -i 's,image: .*,image: '"${IMAGE_REPO}/brancz/kube-rbac-proxy:${KUBE_RBAC_PROXY_LATEST_TAG}"',' ./test/infrastructure/docker/config/manager/manager_auth_proxy_patch.yaml
```

Comment:
did they pull kube-rbac-proxy from capd?

Reply:
yup, as a part of this PR https://github.com/kubernetes-sigs/cluster-api/pull/4640/files